react/components/Table/README.md

```diff
@@ -56,17 +56,17 @@ const defaultSchema = {
 The Schema property is a JSON used to define the table columns and how they should behave visually. The Schema has properties and each one of them defines a column in the table.
 Example with simple structure:
 
-```md
+```python
```

Comment:
Why not `js`?

Reply:
It's a hack to display the code. This is not living code.
![Screen Shot 2020-05-08 at 17 09 24](https://user-images.githubusercontent.com/2573602/81445113-afbb8480-914e-11ea-9763-8adbf4de7fae.png)

Reply:
Maybe adding `static` on it?

https://github.com/styleguidist/react-styleguidist/blob/master/docs/Documenting.md#usage-examples-and-readme-files

Reply:
Good one. Less hacky, i did not know about that one.